doc/proxy-clients.rst

```diff
@@ -97,12 +97,28 @@ You can also pass some options to the Varnish client:
 * ``tags_header`` (X-Cache-Tags): Allows you to change the HTTP header used for
   tagging. If you change this, make sure to use the correct header name in your
   :doc:`proxy server configuration <proxy-configuration>`;
+* ``tags_purge_header`` (key): Allows you to change the HTTP header used for
+  purging tags when ``tag_mode`` is set to one of the ``purge*`` options. If
+  you change this, make sure to use the correct header name in your
+  :doc:`proxy server configuration <proxy-configuration>`;
+* ``tag_mode`` (ban): Makes it possible to invalidate tags
+  using purge when for instance using ``xkey`` instead of ban. Supports three
+  modes: ``ban``, ``purge`` and ``purge_single`` for purge per tag.
```

Comment:
The difference between purge and purge single is somewhat important since xkey only takes one key at a time on purge atm, and this is where `purge_single` comes in.  `purge` is doing it like ban, just limited by header length, but this won't work with xkey before a PR over there is merged and new release trickles down to distributions.

See: https://github.com/FriendsOfSymfony/FOSHttpCache/pull/332#issuecomment-264200486

ref